src/StdToml.sol

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.9.0;
+
+pragma experimental ABIEncoderV2;
+
+import {VmSafe} from "./Vm.sol";
+
+// Helpers for parsing and writing TOML files
+// To parse:
+// ```
+// using stdToml for string;
+// string memory toml = vm.readFile("some_path");
+// toml.parseUint("<json_path>");
+// ```
+// To write:
+// ```
+// using stdToml for string;
+// string memory toml = "[key]\n a = 1\n b = 2";
+// toml.serialize("key", "c", 3);
```

Comment:
just curious why the example here is a bit different/shorter than the one in the StdJson.sol comments?

Reply:
Example code was incorrect, please see the updated example